apps/vr-tests-react-components/src/stories/Spinner.stories.tsx

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28249&commitId=ecf210a62ec01cc770559bb0ee108e0f1c4cb0c4&env=prod&clientType=FLUENTUI)


  <details><summary><b>Spinner converged</b> 163 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Inverted - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Huge - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Huge - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Huge.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label Above - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label Above - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label Above.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label After - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label After - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label After.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label Before - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label Before - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label Before.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label Below - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label Below - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label Below.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Label.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Large - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Large - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Large.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Small - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Small - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Small.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Tiny - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Tiny - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Extra Tiny.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Large - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Large - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Large.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Medium - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Medium - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Medium.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Small - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Small - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Small.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Tiny - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Tiny - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Inverted with Size Tiny.rest.chromium.png|0| Added|
|Spinner converged.Inverted.rest.chromium.png|0| Added|
|Spinner converged.Primary - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Huge - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Huge - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Huge.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label Above - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label Above - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label Above.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label After - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label After - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label After - RTL.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label After.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label Before - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label Before - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label Before - RTL.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label Before.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label Below - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label Below - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label Below.rest.chromium.png|0| Added|
|Spinner converged.Primary with Label.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Large - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Large - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Large.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Small - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Small - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Small.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Tiny - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Tiny - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Extra Tiny.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Large - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Large - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Large.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Medium - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Medium - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Medium.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Small - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Small - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Small.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Tiny - Dark Mode.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Tiny - High Contrast.rest.chromium.png|0| Added|
|Spinner converged.Primary with Size Tiny.rest.chromium.png|0| Added|
|Spinner converged.Primary.rest.chromium.png|0| Added|
|Spinner converged.Inverted - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Huge - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Huge - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Huge.chromium.png|0| Removed|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Above - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Above - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Above.chromium.png|0| Removed|
|Spinner converged.Inverted with Label After - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Label After - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Label After.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Before - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Before - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Before.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Below - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Below - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Label Below.chromium.png|0| Removed|
|Spinner converged.Inverted with Label.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Large - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Large - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Large.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Small - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Small - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Extra Small.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Large - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Large - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Large.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Medium - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Medium - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Medium.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Small - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Small - High Contrast.chromium.png|0| Removed|
|Spinner converged.Inverted with Size Small.chromium.png|0| Removed|
|Spinner converged.Inverted.chromium.png|0| Removed|
|Spinner converged.Primary - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Huge - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Huge - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Huge.chromium.png|0| Removed|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Label Above - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Label Above - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Label Above.chromium.png|0| Removed|
|Spinner converged.Primary with Label After - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Label After - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Label After - RTL.chromium.png|0| Removed|
|Spinner converged.Primary with Label After.chromium.png|0| Removed|
|Spinner converged.Primary with Label Before - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Label Before - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Label Before - RTL.chromium.png|0| Removed|
|Spinner converged.Primary with Label Before.chromium.png|0| Removed|
|Spinner converged.Primary with Label Below - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Label Below - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Label Below.chromium.png|0| Removed|
|Spinner converged.Primary with Label.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Large - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Large - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Large.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Small - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Small - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Extra Small.chromium.png|0| Removed|
|Spinner converged.Primary with Size Large - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Large - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Large.chromium.png|0| Removed|
|Spinner converged.Primary with Size Medium - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Medium - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Medium.chromium.png|0| Removed|
|Spinner converged.Primary with Size Small - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Small - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Small.chromium.png|0| Removed|
|Spinner converged.Primary with Size Tiny - Dark Mode.chromium.png|0| Removed|
|Spinner converged.Primary with Size Tiny - High Contrast.chromium.png|0| Removed|
|Spinner converged.Primary with Size Tiny.chromium.png|0| Removed|
|Spinner converged.Primary.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>